src/main/java/frc/robot/maps/Tempest.java

```diff
@@ -0,0 +1,147 @@
+package frc.robot.maps;
+
+import com.chopshop166.chopshoplib.outputs.MockSpeedController;
+import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
+import com.chopshop166.chopshoplib.sensors.Lidar;
+import com.chopshop166.chopshoplib.sensors.MockPotentiometer;
+import com.chopshop166.chopshoplib.sensors.PIDGyro;
+import com.chopshop166.chopshoplib.sensors.SparkMaxCounter;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.I2C.Port;
+import edu.wpi.first.wpilibj.interfaces.Potentiometer;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import frc.robot.RobotMap;
+
+public class Tempest implements RobotMap {
+
+    @Override
+    public LiftMap getLiftMap() {
+        return new LiftMap() {
+
+            @Override
+            public DigitalInput getUpperLimit() {
+                return new DigitalInput(4);
+            }
+
+            @Override
+            public CANSparkMax getMotor() {
+                // return new MockSpeedController();
+                return null;
```

Comment:
This seems like it should crash if you use Tempest and try to use the lift...

Reply:
No better/ easier way to do. (SadCat)

Reply:
Unfortunately, I know... At least we know where a NullPointerException might occur